src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -48,6 +50,10 @@ def close(self) -> None:
         pass
 
 
+class SiloDataInterface(pydantic.BaseModel):
```

Comment:
With us renaming most of the `Api*` classes to `Rpc*` should this class name also use `Rpc`?